### ISSUE TITLE
House price prediction code 

### ISSUE THREAD

**Issue by anuj13496** [anuj13496/Anuj#1]
# Importing necessary libraries
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LinearRegression
from sklearn.metrics import mean_squared_error, r2_score

# Load the dataset (assuming you have a CSV file named 'house_data.csv' with features and target column)
data = pd.read_csv('house_data.csv')

# Splitting the data into features (X) and target (y)
X = data.drop('house_price', axis=1)
y = data['house_price']

# Splitting the data into training and testing sets (80% train, 20% test)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Creating a Linear Regression model
model = LinearRegression()

# Training the model
model.fit(X_train, y_train)

# Making predictions on the test set
y_pred = model.predict(X_test)

# Evaluating the model
mse = mean_squared_error(y_test, y_pred)
r2 = r2_score(y_test, y_pred)

print(f"Mean Squared Error: {mse}")
print(f"R-squared: {r2}")

# You can now use the trained model to predict house prices for new data.
